girder/molecules/molecules/calculation.py

```diff
@@ -189,6 +189,9 @@ def stream():
 
     @access.public
     def get_calc_cube(self, id, mo, params):
+        if 'status' in params:
+            raise RestException('Invalid JSON or MO.', params['status'])
```

Comment:
What is the is check for?

Reply:
Currently the orbital callback function sends a notification back to the client that includes a status code if the calculation did not succeed. The client makes another request to the endpoint once it gets the notification, so this is checking to see if a status code had been returned or not before bothering to see if the orbital has been cached.